Apps/WebClient/src/ClientApp/src/components/modal/immunizationCard.vue

```diff
@@ -327,8 +356,8 @@ div[class*=" row"] {
     border-radius: 15px 0px 0px 15px;
 
     .left-pane {
-        max-width: 75px;
-        padding: 20px;
+        max-width: 65px;
```

Comment:
new yyyy-MMM-dd (i.e. 2021-Jan-06) display format makes the date showing on 2 lines instead of 1 line on small small such as iPhone6. These css changes fix the issue.